src/explore-education-statistics-common/src/modules/table-tool/components/Footnote.tsx

```diff
@@ -15,14 +15,10 @@ const Footnote = ({ content }: FootnoteProps) => {
       </li>
     );
   });
+
   return (
     <ol className="govuk-list">
-      {footnoteList.slice(0, 3)}
-      {footnoteList.length > 3 && (
-        <Details summary="View additional footnotes">
-          <ol className="govuk-list">{footnoteList.slice(3)}</ol>
-        </Details>
-      )}
+      <CollapsibleList cssDriven>{footnoteList}</CollapsibleList>
```

Comment:
This doesn't look like it would be semantically correct HTML. I think at least one `div` will be a child of the `ol`. 

I'm skeptical on using `CollapsibleList` here as it could be easier to have a prop on `Details` that removes the `display: none` for print media.

Reply:
We are using the CollapsibleList because we need it to expand on print and also the screenreaders need to be able to read the items that are not shown